### PR TITLE
feat: double comment and bangumi addon docs

### PR DESCRIPTION
**Pull request by yixiaojiu**
### Description

image

### Linked Issues

https://github.com/YunYouJun/valaxy/issues/310

### CODE REVIEW

File: packages/valaxy-theme-yun/components/YunComment.vue [hunk 1]
```diff
@@ -1,14 +1,34 @@
 <script lang="ts" setup>
 import { useRuntimeConfig } from 'valaxy'
+import { ref } from 'vue'
 
+const activeComment = ref<'waline' | 'twikoo'>('waline')
 const runtimeConfig = useRuntimeConfig()
+const isDoubleComment = !!(runtimeConfig.value.addons['valaxy-addon-waline'] && runtimeConfig.value.addons['valaxy-addon-twikoo']) as boolean
+
+function toggleComment() {
+  activeComment.value = activeComment.value === 'waline' ? 'twikoo' : 'waline'
```
Comment (YunYouJun) [inline packages/valaxy-theme-yun/components/YunComment.vue]: Known comment systems include waline/twikoo/artalk/gitalk/utterance/giscus/disqus and others.

Although we are not currently developing other comment addons.
I think we should reserve that logic.

Maybe we can use an array and `includes`.

File: packages/valaxy-theme-yun/components/third/YunTwikoo.vue [hunk 2]
```diff
@@ -2,9 +2,14 @@
 import * as addonTwikoo from 'valaxy-addon-twikoo'
 import 'valaxy-addon-twikoo/client/styles/index.scss'
 import { isEmptyAddon } from 'valaxy'
+import { onBeforeUnmount } from 'vue'
 
 if (!isEmptyAddon(addonTwikoo))
   addonTwikoo.useTwikooWithOptions()
+
+onBeforeUnmount(() => {
+  document.querySelector('#twikoo')?.remove()
```
Comment (YunYouJun) [inline packages/valaxy-theme-yun/components/third/YunTwikoo.vue]: Why we need this?

**Review by YunYouJun — CHANGES_REQUESTED** [review]

File: packages/valaxy-theme-yun/components/third/YunTwikoo.vue (hunk 2 above)
Comment (yixiaojiu) [reply packages/valaxy-theme-yun/components/third/YunTwikoo.vue]: twikoo will replace `<div id="tcomment" w="full" />` to `<div id="twikoo" class="twikoo"></div>`. When component unmount, `<div id="twikoo" class="twikoo"></div>` isn't removed correctly.

File: packages/valaxy-theme-yun/components/YunComment.vue (hunk 1 above)
Comment (yixiaojiu) [reply packages/valaxy-theme-yun/components/YunComment.vue]: Wow, There so many comment systems. How about replace Switch with Select and add a configuration like this?

```ts
export default defineValaxyConfig({
  siteConfig: {
    comment: {
      enable: true,
      includes: ['valaxy-addon-twikoo', 'valaxy-addon-waline']
    },
  },
})
```

**Comment by yixiaojiu** [comment]
image

UI is simple.

**Pull request merged**